IOTATransferRestRequestTemplate.py

```diff
@@ -128,17 +128,22 @@ def requestData():
     request_id = register_request().bytes
     transaction = create_transaction_dictionary(request_tag=request_id)
 
-    # :todo: Populate these values and/or make API object globally-accessible.
-    api = create_iota_object(uri=uri, auth_token=auth_token, seed=seed)
-    api.send_transfer(
-        depth=transaction['depth'],
+    if transaction['tag'] == Tag(request_id):
```

Comment:
Moved the conditional way higher in the function; if `transaction['tag`]` doesn't match `request_id`, then it doesn't seem like there would be any point is calling `send_transfer`, since we wouldn't do anything with the result anyway.